### PR TITLE
RC custom updates test

### PR DESCRIPTION
**Pull request by packaging-art**
Take a look:

// Variables

// Sitewide Typography
// Breadcrumbs
// Buttons
// Cards
// Dropdowns
// Forms
// Navigation
// Pagination

I have annotated in-line as much as I could + explained in the commits.

This gets us virtually 100% to the mock-up on a lot of components, specifically navigation, dropdown, and breadcrumbs can be built virtually 1 for 1 now.

Take a look this when you can and lets chat. I should be available tomorrow for a bit.

### CODE REVIEW

File: scss/_rc-custom-adjustments.scss [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+// Cards
+// Dropdowns
+// Forms
+// Navigation
+// Pagination
+
+
+//Variables
+
+$box-shadow: 0 0 13px 0 rgba($black,.15);
```
Comment (anorborg) [inline scss/_rc-custom-adjustments.scss]: Looks like a new variable so lets stick with the `rc-` prefix for custom vars

File: scss/_rc-custom-adjustments.scss [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+
+
+
+// Sitewide Typography
+
+body {
+  // Improves sitewide typogrpahy legibility by using built in kerning (letterspacing math, ex spacing between Wa combination) of the typeface, resulting in more readable and overall better looking text.
+  text-rendering: optimizeLegibility;
```
Comment (anorborg) [inline scss/_rc-custom-adjustments.scss:30]: I assume that edge and IE not supporting this isn't a big deal?
https://caniuse.com/#search=text-rendering

File: scss/_rc-custom-adjustments.scss [hunk 3]
```diff
@@ -0,0 +1,180 @@
...
+
+  // Activates improved rendering of typography on webkit based browsers, making text closer to its intended look, as it was designed and appears in print.
+  -webkit-font-smoothing: antialiased;
```
Comment (anorborg) [inline scss/_rc-custom-adjustments.scss:33]: again, keep cross-browser in mind with these advanced values.

https://developer.mozilla.org/en-US/docs/Web/CSS/font-smooth

**Review by anorborg — CHANGES_REQUESTED** [review]
Navs don't like like the mockups as far as I can see either?

capture 2017-06-25 at 6 04 01 pm

**Comment by packaging-art** [comment]
going to guess you need to compile the `npm run css` in order to get the contents of the new css file

screen shot 2017-06-25 at 9 59 40 pm

File: scss/_rc-custom-adjustments.scss (hunk 1 above)
Comment (packaging-art) [reply scss/_rc-custom-adjustments.scss]: agreed — will do

File: scss/_rc-custom-adjustments.scss (hunk 2 above)
Comment (packaging-art) [reply scss/_rc-custom-adjustments.scss:30]: no issue, it progressive enhancement. IE is good already. browsers that support it get even better. no user is hurt.

File: scss/_rc-custom-adjustments.scss (hunk 3 above)
Comment (packaging-art) [reply scss/_rc-custom-adjustments.scss:33]: ditto, as above. Mozilla refuses to implement this, so they get chunkier looking fonts. still good. but webkit user benefit for text that is a little nicer.

Comment (anorborg) [reply scss/_rc-custom-adjustments.scss:33]: I do see some suggestions to use `-moz-osx-font-smoothing` in mozilla:

https://medium.com/@MateMarschalko/improving-font-rendering-with-css-3383fc358cbc

**Comment by anorborg** [comment]
Looks like I forgot to get latest (I already had a earlier version of the branch when you asked me to check it out. Navbar is updated. Looks like no changes to form labels though?

**Comment by packaging-art** [comment]
form labels? placeholder is bold and there is less padding-left. looks right to me

screen shot 2017-06-26 at 10 28 51 am

File: scss/_rc-custom-adjustments.scss (hunk 3 above)
Comment (packaging-art) [reply scss/_rc-custom-adjustments.scss:33]: good, catch — added

**Comment by anorborg** [comment]
I was referring to this?

image

**Comment by anorborg** [comment]
Also the placement is for the vertical forms, but I figured the label styles would match.

**Comment by packaging-art** [comment]
gotcha — latests commits takes care of that + the error and success messages. good eye again, missed that part.

**Review by anorborg — APPROVED** [review]
:shipit:

**Pull request merged**